Remotion/ObjectBinding/UnitTests/ObjectBinding.UnitTests.csproj

```diff
@@ -12,6 +12,7 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="..\..\..\SharedSource\Development\NUnit\UnitTesting\*.cs" Link="SharedSource\NUnit\UnitTesting\%(Filename)%(Extension)" />
+    <Compile Include="..\..\..\SharedSource\Development\Globalization\*.cs" Link="SharedSource\NUnit\Globalization\%(Filename)%(Extension)" />
```

Comment:
THere's a different include pattern in line 14. I think this needs chaing in line 15?
-> `SharedSource\NUnit\UnitTesting\Globalization` would be the location for the shared file, also in the sharedsource location.